package.json

```diff
@@ -59,14 +58,11 @@
     "start-exp": "yarn workspace @fluentui/react-experiments start",
     "stats:build": "gulp stats",
     "stats:save": "gulp stats:save",
-    "sync:beachball": "beachball sync --scope \"!packages/fluentui/*\"",
     "test": "lage test --verbose",
     "test:fluentui:circulars": "gulp test:circulars:run",
     "test:fluentui:e2e": "yarn workspace @fluentui/e2e test",
     "test:fluentui:projects": "yarn workspace @fluentui/projects-test test",
-    "update-api": "echo API files are now updated when running yarn build",
-    "update-snapshots": "lage update-snapshots --verbose",
-    "vrtest": "cd apps && cd vr-tests && yarn screener"
```

Comment:
we have multiple vrtest apps now, removed as this is confusing to invoke - preferred way is to invoke commands from packages via `yarn workspace`